src/core/evaluator.js

```diff
@@ -217,6 +217,50 @@ var PartialEvaluator = (function PartialEvaluatorClosure() {
     }
   };
 
+  // Convert PDF blend mode names to HTML5 blend mode names.
+  function normalizeBlendMode(value) {
+    if (!value || !value.name) {
```

Comment:
This might be paranoid, but can you please do `if (!value || !isName(value)) {` or simply `if(!isName(value)) {` instead, to prevent any possible issues here?